### PR TITLE
add micropython-stubs

### PR DESCRIPTION
**Pull request by Josverl**
- Add micropython-stubs 
- update micropython-stubber description
- add Micropython-magic 

for `micropython-magic` I added an additional category ## Jupyter as im pretty sure there are a few other ipython/jupyter integrations out there that might be added at some time.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -935,7 +935,8 @@ input via push buttons or a navigation joystick and an optional rotary encoder.
 ### Code Generation
 
 * [micropy-cli](https://github.com/BradenM/micropy-cli) - Micropy CLI is a project management/generation tool for writing MicroPython code in modern IDEs such as Visual Studio Code.
-* [micropython-stubber](https://github.com/Josverl/micropython-stubber) - Generate and use stubs for different MicroPython firmwares to use with Visual Studio Code and/or Pylint.
+* [micropython-stubber](https://github.com/Josverl/micropython-stubber) - Generate and use stubs for different MicroPython firmwares to use with Visual Studio Code or any IDE and linter.
+* [micropython-stubs](https://github.com/Josverl/micropython-stubs) - Stubs of most MicroPython ports, boards and versions to make writing code that much simpler..
```
Comment (andypiper) [inline readme.md:939]: tiny nitpick, double . on the end of the line here

File: readme.md [hunk 2]
```diff
@@ -980,6 +980,9 @@ input via push buttons or a navigation joystick and an optional rotary encoder.
 * [mpsync](https://github.com/thilomichael/mpsync) - A tool that automatically synchronizes code to a MicroPython board.
 * [mpremote](https://github.com/micropython/micropython/blob/master/tools/mpremote/README.md) - Powerful official shell that supports mounting the host's current directory on the target. Run code without changing the target's filesystem.
 
+#### Jupyter
+* [micropython-magic](https://github.com/josverl/micropython-magic) - A file system bridge to synchronize and manage files on a device running MicroPython.
```
Comment (andypiper) [inline readme.md:984]: I don't think this is the correct description (looks like a cut-paste from another entry which has not had the description updated)

**Review by andypiper — CHANGES_REQUESTED** [review]
Could you update the PR to resolve the -magic description and tidy the extra dot in the other update? Thanks.

File: readme.md (hunk 1 above)
Comment (mcauser) [reply readme.md:939]: I've fixed

File: readme.md (hunk 2 above)
Comment (mcauser) [reply readme.md:984]: Yep, copy pasta. I've grabbed the latest desc from Github and committed a fix

**Comment by mcauser** [comment]
Merged, fixed, thanks @Josverl and @andypiper

**Comment by Josverl** [comment]
:-) thanks - apparently I was too focused on not upsetting the structure - and forgot to copy in the correct description.
Andy, sorry for missing the review requests , and thanks Mike for the fix

**Pull request merged**